packages/compass-export-to-language/src/stores/store.js

```diff
@@ -10,6 +10,31 @@ import {
 } from '@mongodb-js/mongodb-redux-common/app-registry';
 import reducer from '../modules';
 
+// TODO: replace this with state coming from the right layer.
+// this kind of information should not be derived
+// from dataService as it operates on a lower level,
+// as a consequence here we have to remove the `appName` that only
+// the dataService should be using.
+function getCurrentlyConnectedUri(dataService) {
```

Comment:
This was moved from compass collection where it was before to the only place that actually needs this state from dataService